### PR TITLE
[VEN-1795] OZ Diamond Comptroller audit fixes

### PR DESCRIPTION
**Pull request by Debugger022**
## Description

Resolves #1795

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

File: contracts/Comptroller/Diamond/facets/SetterFacet.sol [hunk 1]
```diff
@@ -324,7 +324,7 @@ contract SetterFacet is FacetBase {
      * @param paused_ The new paused state (true=paused, false=unpaused)
      */
     function _setActionsPaused(address[] calldata markets_, Action[] calldata actions_, bool paused_) external {
-        ensureAllowed("_setActionsPaused(address[],uint256[],bool)");
```
Comment (chechu) [inline contracts/Comptroller/Diamond/facets/SetterFacet.sol:321]: If we do this we should authorized the Normal Timelock in the ACM to execute this new function. We should do this in the VIP associated to this deployment

File: contracts/Comptroller/Diamond/facets/FacetBase.sol [hunk 2]
```diff
@@ -18,13 +18,13 @@ contract FacetBase is ComptrollerV12Storage, ExponentialNoError {
...
+    uint224 public constant VENUS_INITIAL_INDEX = 1e36;
     // closeFactorMantissa must be strictly greater than this value
-    uint internal constant closeFactorMinMantissa = 0.05e18; // 0.05
+    uint256 internal constant CLOSE_FACTOR_MIN_MANTISSA = 0.05e18; // 0.05
     // closeFactorMantissa must not exceed this value
-    uint internal constant closeFactorMaxMantissa = 0.9e18; // 0.9
+    uint256 internal constant CLOSE_FACTOR_MAX_MANTISSA = 0.9e18; // 0.9
     // No collateralFactorMantissa may exceed this value
-    uint internal constant collateralFactorMaxMantissa = 0.9e18; // 0.9
+    uint256 internal constant COLLATERAL_FACTOR_MAX_MANTISSA = 0.9e18; // 0.9
```
Comment (chechu) [inline contracts/Comptroller/Diamond/facets/FacetBase.sol:27]: This seems like a breaking change that can impact third parties.The public function `closeFactorMinMantissa()`, for example, will disappear

Comment (chechu) [reply contracts/Comptroller/Diamond/facets/FacetBase.sol:27]: So, I think I wouldn't do this change, to be conservative

File: contracts/Comptroller/Diamond/facets/SetterFacet.sol (hunk 1 above)
Comment (Debugger022) [reply contracts/Comptroller/Diamond/facets/SetterFacet.sol:321]: I have updated the VIP for the diamond proxy: https://github.com/VenusProtocol/vips/pull/11/commits/6cdc98b1677e6e0c8abfdc2726515b4e2063abfd

File: contracts/Comptroller/Diamond/facets/FacetBase.sol (hunk 2 above)
Comment (Debugger022) [reply contracts/Comptroller/Diamond/facets/FacetBase.sol]: Reverted changes for N-08, fd07edd871d4f1199e6676910351b6d2f8c5a760

**Review by chechu — APPROVED** [review]

File: contracts/Comptroller/Diamond/facets/FacetBase.sol (hunk 2 above)
Comment (chechu) [reply contracts/Comptroller/Diamond/facets/FacetBase.sol]: You were right @0xlucian, only the public `venusInitialIndex` variable was affected. I prefer to revert it (as @Debugger022 did)

**Pull request merged**